### PR TITLE
Bootstrap feedback cycle

### PR DESCRIPTION
**Pull request by sternhenri**
Catch up re this [set of notes](https://docs.google.com/document/d/1pDcme0pHUKHEc39OQn95huU17WyIb4G8XIHjz15Fx28/edit#) (only available to a few folks). Main action items:
- de-goify code snippets
- mark default values as such (vs mandatory)
- add some details in various places

### CODE REVIEW

File: bootstrap.md [hunk 1]
```diff
@@ -33,13 +33,17 @@ We leave trustless initial peer selection as future work.
...
+
+In order to secure the network, these "trusted nodes" must validate incoming blocks they receive, running as full nodes  on the network on a voluntary rather than economic basis (without participating in consensus).
 
 We leave trustless (or fully decentralized) bootstrapping for future work.
 
 A strawman bootstrapping sequence would have nodes use these initial peers to sync up to the canonical chain. However, new connection requests might effecitvely DoS these peers, making the Filecoin network unavailable to new nodes. Therefore, Filecoin bootstrapping seeks to "load-balance" initial requests to ensure the network remains widely available.
 
-We connect to each of these initial peers and then initiate the `hello handshake` (see in [network protocols](./network-protocols.md#hello-handshake)), getting back
+We connect to each of these initial peers (via DNS or IP, with a slight preference to IP connections as they are harder to coop) and then initiate the `hello handshake` (see in [network protocols](./network-protocols.md#hello-handshake)), getting back
```
Comment (whyrusleeping) [inline bootstrap.md]: i'd specify 'hard coded IP', as you'll likely still use IP even if you use DNS to resolve it first.

**Comment by whyrusleeping** [comment]
@sternhenri any update here?

**Comment by sternhenri** [comment]
sorry. filing shortly.

**Comment by sternhenri** [comment]
@whyrusleeping last step here for me (imo) is to de-go-ify the code snippets here, and have the spec be a bit less normative in terms of how to do things. 

Do you want me to move ahead with this, or should I drop this in favor of https://github.com/filecoin-project/specs/pull/295 which achieves this purpose?

**Comment by sternhenri** [comment]
@whyrusleeping done with this. 

Lmk if you'd rather merge w. code ([slightly degoified](https://github.com/filecoin-project/specs/pull/269/commits/e35a4cdb632c86983db0d15dd68b25913c32c3e6)) or w.o. ([high-level pseudocode](https://github.com/filecoin-project/specs/pull/269/commits/8a6a8573b36d203ad74fbd8e4522118ff4ec2af4))

**Comment by dignifiedquire** [comment]
I like the "high-level pseudocode" much more

**Comment by sternhenri** [comment]
Ready to go then :D

**Comment by sternhenri** [comment]
Too far from master at this point. Choice was to leave bootstrap a bit more up to implementation. Closing this out accordingly.

**Pull request closed without merging**